docs/_index.md

```diff
@@ -44,6 +44,7 @@ To learn more about Cortex, consult the following talks and articles.
 
 ### Recent talks and articles
 
+- Dec 2020 blog post "[How AWS and Grafana Labs are scaling Cortex for the cloud](https://aws.amazon.com/blogs/opensource/how-aws-and-grafana-labs-are-scaling-cortex-for-the-cloud/)
```

Comment:
Please do the same change to `README.md` too. Unfortunately it's currently a copy-paste.

Reply:
Good call. Thanks Marco